src/sentry/south_migrations/0288_set_release_project_new_groups_to_zero.py

```diff
@@ -0,0 +1,708 @@
+# -*- coding: utf-8 -*-
+from south.utils import datetime_utils as datetime
+from south.db import db
+from south.v2 import DataMigration
+from django.db import models
+
+class Migration(DataMigration):
+
+    def forwards(self, orm):
+        "Write your forwards methods here."
+        modified = True
```

Comment:
We should commit the transaction before doing this. So this can run outside of a transaction. Same as you did in the last one.